sdk/pom.xml

```diff
@@ -311,6 +305,7 @@
                     <include>schnorr_sk00_hex</include>
                     <include>sample_proving_key_7_keys_with_threshold_5</include>
                     <include>sample_vk_7_keys_with_threshold_5</include>
+					<include>pippo_vk</include>
```

Comment:
can be removed.
And moreover, let's have a regression test for proof verification.